CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# 🤝***First time contributing? We will help you out.*** 👍
+
+![GitHub custom open for collaboration](https://img.shields.io/badge/Open%20For-Collaboration-brightgreen?style=for-the-badge)
+
+Refer to the following articles on the basics of Git and Github and can also contact the Project Mentors, in case you are stuck:
```

Comment:
```suggestion
Refer to the following articles on the basics of Git and Github. You can also contact the Project Mentors, in case you are stuck:
```

Reply:
I have corrected it. Thank you for pointing it out